### PR TITLE
Feature/unlock screen UI

### LINKED ISSUE #16: Implement Unlock Screen
Should be  a password field 
Viewport should be bigger

### PR DESCRIPTION
**Pull request by chris-aeviator**
image
closes #16

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,8 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+yarn.lock
```
Comment (bumi) [inline .gitignore:13]: hmm. shouldn't the lock files be in git to define exact current dependencies?

Comment (chris-aeviator) [reply .gitignore:13]: oh well - ur right

File: src/app/components/Unlock/index.jsx [hunk 2]
```diff
@@ -33,20 +35,26 @@ class Unlock extends React.Component {
 
   render() {
     return (
-      <div>
+      <div class="unlock--container">
+        <UnlockTwoTone
+          className="unlock--container__icon"
+          twoToneColor="#1890ff"
```
Comment (bumi) [inline src/app/components/Unlock/index.jsx:41]: Do you think we can/should define the colors somehow globally in variables?

File: src/app/components/Unlock/index.jsx [hunk 3]
```diff
@@ -33,20 +35,26 @@ class Unlock extends React.Component {
 
   render() {
     return (
-      <div>
+      <div class="unlock--container">
```
Comment (bumi) [inline src/app/components/Unlock/index.jsx:38]: do we follow some kind of naming convention here? 
what are those?

**Review by bumi — APPROVED** [review]

File: src/app/components/Unlock/index.jsx (hunk 3 above)
Comment (chris-aeviator) [reply src/app/components/Unlock/index.jsx:38]: It's a scheme/ convention called BEM (Block, Element, Modifier)
http://getbem.com/introduction/

File: .gitignore (hunk 1 above)
Comment (chris-aeviator) [reply .gitignore:13]: removed it in https://github.com/bumi/lightning-browser-extension/pull/41/commits/f7e4da5f5452bf72b1a39ee6634f04287fc27ee0

File: src/app/components/Unlock/index.jsx (hunk 2 above)
Comment (chris-aeviator) [reply src/app/components/Unlock/index.jsx:41]: fixed in https://github.com/bumi/lightning-browser-extension/pull/41/commits/d92f9576f087d16e5d86ad96c0894d76582c339a

**Pull request merged**